content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -62,270 +62,117 @@ sleep.legacy to httpbin.bar: 200
 sleep.legacy to httpbin.legacy: 200
 {{< /text >}}
 
-You should also verify that there is a default mesh authentication policy in the system, which you can do as follows:
+Verify there is no peer authentication policy in the system with the following command:
 
 {{< text bash >}}
-$ kubectl get policies.authentication.istio.io --all-namespaces
+$ kubectl get peerauthentication --all-namespaces
 No resources found.
 {{< /text >}}
 
-{{< text bash >}}
-$ kubectl get meshpolicies.authentication.istio.io
-NAME      AGE
-default   3m
-{{< /text >}}
-
 Last but not least, verify that there are no destination rules that apply on the example services. You can do this by checking the `host:` value of
  existing destination rules and make sure they do not match. For example:
 
 {{< text bash >}}
 $ kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"
-    host: istio-policy.istio-system.svc.cluster.local
-    host: istio-telemetry.istio-system.svc.cluster.local
 {{< /text >}}
 
 {{< tip >}}
 Depending on the version of Istio, you may see destination rules for hosts other then those shown. However, there should be none with hosts in the `foo`,
 `bar` and `legacy` namespace, nor is the match-all wildcard `*`
 {{< /tip >}}
 
-## Globally enabling Istio mutual TLS
+## Auto mTLS
```

Comment:
```suggestion
## Auto mutual TLS
```

Reply:
I've checked and most spelling errors are caused because we don't abbreviate mutual TLS. 
Please replace all instances of "mTLS" with "mutual TLS"